### DISCUSSION TITLE
retrieve records from BigQueryGetDataOperator

### DISCUSSION THREAD

**Discussion by shuka99** [apache/airflow#26403] [Q&A]
Dear all,
In airflow I use BigQueryGetDataOperator to get 1 row of 3 columns from a table.
def SET_flags(flags):
	return flags
get_flags = BigQueryGetDataOperator (
.............
)
I retrieve the values as
flags_str = SET_flags("ti.xcom_pull('get_flags')")
The value returned is in the form [['aaa','bbb','ccc']] which is a string(checked the type by printing type()

I want to retrieve the individual values for which i'm converting it to a list and try to retrieve list[0],list[1] etc:

flags_list = flags_str.strip("][").replace("'","").split(',')

This works in python but within airflow list returns ['()']
Could anyone please help me how to get individual values in airflow.
Thank
Sug

**Comment by potiuk** [comment]
I think you should look at the actual type returned by xcom _pull and debug it.  Looks like the value should be array. this is what GetDataOperator returs:
```
        table_data = [row.values() for row in rows]
        return table_data
```

But in any case you should just debug it and check it yourself and make sure you got the typing correct. This is just basic python debugging
